_posts/2020-12-26-how-to-use-iphone-as-webcam.md

```diff
@@ -3,7 +3,8 @@ layout: post
 title:  "How to use an iPhone as a webcam (EpocCam Review)"
 author: ogjunkyard
 categories: [ webcam ]
-image: "assets/images/pink-iphone.jpg"
+image-location: "assets/images/"
```

Comment:
Move to `_config.yml`.

Reply:
Since this has been moved to _config.yml, do we still need this setting in the post?